.github/release.yml

```diff
@@ -14,3 +14,9 @@ changelog:
     - title: Internal Improvements
       labels:
         - Internal
+    - title: Memory Optimizations
+      labels:
+        - memory
```

Comment:
```suggestion
        - Internal
        - memory
```

Can't it be added into as a internal improvement, instead of separate section?

Reply:
Oh sorry I thought you meant you wanted it as a section, my bad

Reply:
:sweat_smile: On second thought just remove the memory label here, and add Internal where needed?

Reply:
It won't hurt to add it to the internal improvements section, ill put it there for now and if we don't use it its not a big deal. We may want to categorise some things as memory in future